### ISSUE TITLE
No files found to lint on Node.js project

### ISSUE THREAD

**Issue by ilyavolodin** [madskristensen/WebAnalyzer#4]
I have created a new Node.js project (using Node.js tools for Visual Studio) and when I right click on the folder in solution explorer and tell it to run Web Analyzer, I get a message "No files found to lint" even though the folder if full of `.js` files. Folder is marked as (node) in solution explorer.
Running Visual Studio 14.0.23107 and latest WebAnalyzer. Let me know if you need more information.

**Comment by madskristensen** [comment]
I haven't tested it in a Node.js project at all yet. Thanks for the heads up on the issue

**Comment by ilyavolodin** [comment]
I think there's a possibility of WebAnalyzer colliding with Node.js tools. I did a clean install of Visual Studio, then installed Node.js tools and then WebAnalyzer. When I tried to open "Edit ESLint settings (.eslintrc)" from `Tools` menu initially, I was getting an exception, until I did a full reset of WebAnalyzer. So the issue above might be related to that. Or maybe I just have a corrupted installation of extensions somehow.

**Comment by BitSchupser** [comment]
Hi,
I have a similar issue in an Apache Cordova project. I also have Node.js tools installed.
When I open the solution I get the following error from web linter:

```
12.12.2015 16:39:30: System.IO.IOException: Der Prozess kann nicht auf die Datei "node_modules.7z" zugreifen, da sie von einem anderen Prozess verwendet wird.
   bei System.IO.Directory.DeleteHelper(String fullPath, String userPath, Boolean recursive, Boolean throwOnTopLevelDirectoryNotFound)
   bei System.IO.Directory.Delete(String fullPath, String userPath, Boolean recursive, Boolean checkHost)
   bei WebLinter.LinterFactory.<InitializeAsync>d__6.MoveNext() in C:\projects\webanalyzer\src\WebLinter\Linting\LinterFactory.cs:Zeile 96.
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   bei WebLinterVsix.WebLinterInitPackage.<>c.<<Initialize>b__0_1>d.MoveNext() in C:\projects\webanalyzer\src\WebLinterVsix\VSPackage.cs:Zeile 68.

´´´

Sorry for the german error message ;-). It basically says that the process cannot acces the file "node_modules.7z" because it is in use by another process.

**Comment by madskristensen** [comment]
This may have been fixed in version 1.5.
